internal/controllers/gateway/httproute_controller.go

```diff
@@ -585,3 +600,101 @@ func (r *HTTPRouteReconciler) getHTTPRouteRuleReason(ctx context.Context, httpRo
 	}
 	return gatewayv1beta1.RouteReasonResolvedRefs, nil
 }
+
+// ensureParentsAcceptedCondition sets the "Accepted" condition of HTTPRoute status.
+func (r *HTTPRouteReconciler) ensureParentsAcceptedCondition(
+	ctx context.Context,
+	httproute *gatewayv1beta1.HTTPRoute,
+	gateways []supportedGatewayWithCondition,
+	conditionStatus metav1.ConditionStatus,
+	conditionReason string,
```

Comment:
ditto about the type:

```suggestion
	conditionReason gatewayv1beta1.RouteConditionReason,
```

Reply:
done